### DISCUSSION TITLE
Specify default configuration in Amiberry.conf

### DISCUSSION THREAD

**Discussion by StusBrainDump** [BlitterStudio/amiberry#1393] [Ideas]
Instead of specifying a default config to use on launch by naming the config as default.uae, allow users to specify a default config via a setting in Amiberry.conf (for example default_config = A1200-WHD)

Naming a config as default.uae isn't very aesthetically pleasing :)

**Comment by giantclambake** [comment]
I consider this to be more the purview of host OS functionality, and not something that needs to be added to amiberry.

The reasons I say that ;

In the desktop usage case, it's trivial to create a desktop launcher/icon, for the purposes of launching into the emulation, using one's preferred config.uae file...(I can create such in roughly 30seconds).

On the cmdline, it's just as easy....ie; lets say I prefer to start my ClassicWB-P96 setup...I would;

```
cat > ~/classicwb << "EOF"
exec /home/gcb/Amiberry/amiberry conf/ClassicWB-P96.uae
EOF
```
//mark the file executable 

`chmod 0755 ~/classicwb`

Then if I'm in my $HOMEDIR I can just type  ./classicwb  to launch amiberry direct to emulation, using the specified config.uae file.

Although it's been more than a decade since I used macOS, and for sure it'll be a little different, but I believe it's just as easy/simple to do the same things in that OS as it is in linux, and if that's the case, adding such an option to amiberry.conf is unnecessary ; we already have `default.uae` for that =)
